pkg/asset/agent/joiner/addnodesconfig.go

```diff
@@ -123,6 +124,9 @@ func (a *AddNodesConfig) finish() error {
 	if err := a.validateSSHKey(); err != nil {
 		allErrs = append(allErrs, err...)
 	}
+	if err := ValidateAdditionalNTPSources(field.NewPath("AdditionalNTPSources"), a.Config.AdditionalNTPSources); err != nil {
```

Comment:
Does this mean that we're requiring the user to specify it manually in the config?
I think users will expect us to retrieve this info from the cluster automatically.
Unfortunately there is no API in the cluster where this is stored today, so to get this info we'd have to read the MachineConfig resource. Since we only care about getting it into the agent ignition here, maybe we could just apply that MachineConfig (if it exists) to the ignition verbatim to avoid parsing it?
In the near future [METAL-1136](https://issues.redhat.com/browse/METAL-1136) will need to add it in an API somewhere, so that may soon be an option.

Reply:
Yes, I wasn't able to find that piece of configuration stored in the cluster - if not in install-config.yaml copy stored into the `kube-system/cluster-config-v1` cm, but we want to [stay away](https://issues.redhat.com/browse/AGENT-966) from that. 
I saw there was an `openshift-machine-api/50-workers-chrony-configuration` cm, but it contains just the `/etc/chrony.conf` with all the NTP servers, not sure how to filter out the ones defined by the system from the ones defined by the user (originally in the `additionalNTPSources` field)?

Reply:
> Since we only care about getting it into the agent ignition here, maybe we could just apply that MachineConfig (if it exists) to the ignition verbatim to avoid parsing it?

:thinking: Ona secon thought, it sounds like a good idea, we just need to cover the live iso phase, and configuring the same ntp servers (as it will be done for the worker when it will join) should be enough - with the additional bonus of not requiring the user to explicitly configure the servers. Let me try it out

Reply:
(from a quick check it looks like the infraenv.AdditionalNTPSources are required by AS just for day1, so no need to configure it in day2)